sw/airborne/math/pprz_algebra_float.c

```diff
@@ -520,6 +520,33 @@ void float_quat_of_eulers_zxy(struct FloatQuat *q, struct FloatEulers *e)
   q->qz =  s_phi2 * s_theta2 * c_psi2 + c_phi2 * c_theta2 * s_psi2;
 }
 
+/**
+ * @brief quat from euler rotation 'YXZ'
+ * This function calculates a quaternion from Euler angles with the order YXZ,
+ * so pitch, roll, yaw, instead of the conventional ZYX order.
+ * See https://en.wikipedia.org/wiki/Euler_angles
+ *
+ * @param q Quat output
+ * @param e Euler input
+ */
+void float_quat_of_eulers_yxz(struct FloatQuat *q, struct FloatEulers *e)
+{
+  const float phi2   = e->phi / 2.0;
+  const float theta2 = e->theta / 2.0;
+  const float psi2   = e->psi / 2.0;
```

Comment:
it would be better if this was 2.f instead of 2.0 (which is a double)

Reply:
I didn't know this, maybe also good to keep this in mind with the defines in airframe files?

Reply:
Actually, there are a lot more of these numbers even in the same file, change those as well? I did see that for chibios and the F7 the flag -fsingle-precision-constant is defined, which should make those numbers single precision, but I don't know what happens for other configurations.